lib/ansible/modules/network/checkpoint/checkpoint_object_facts.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/python
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'network'}
+
+
+DOCUMENTATION = """
+---
+module: checkpoint_object_facts
+short_description: Get object facts on Checkpoint over Web Services API
+description:
+  - Get object facts on Checkpoint devices.
+    All operations are performed over Web Services API.
+version_added: "2.8"
+author: "Ansible by Red Hat (@rcarrillocruz)"
+options:
+  uid:
+    description:
+      - UID of the object. If UID is not provided. If not provided, it will do a full search
+        which can be filtered with the filter argument.
+  filter_expression:
+    description:
+      - Filter expression for search. It accepts AND/OR logical operators and performs a textual
+        and ip address search. To search only IP address, set ip_only argument to True.
+        which can be filtered with the filter argument.
+  ip_only:
+    description:
+      - Filter only by IP address.
+    type: bool
+    default: False
```

Comment:
```suggestion
    default: false
```

Reply:
We are trying to standardize to true/false (lowercase) to match the strict YAML rules (so linters won't flag this as a warning).